mirte_telemetrix/scripts/mappings/pcb.py

```diff
@@ -86,7 +59,70 @@ def get_max_pwm_value():
     return board_mapping.get_max_pwm_value()
 
 
+def generate_motor_mapping(pin_a, pin_b):
+    # pin a has preference for pwm
+    # This will make sure, dp, pp and dd is always possible when using a connector without knowing what type of control
+    #     	A	B
+    # pp	P1	P2
+    # dp	P1	D1
```

Comment:
With this setup I think we should either:
a) rename dp -> pd (since we have P1, D1)
b) change to D1, P1

I think I prefer a), since otherwise people could get confused with a pd-controller or something. But this will probably also affect the way you have paced the other P1, P2 etc in pp.

Reply:
This mapping is only internal, and we should make sure that A is always capable of pwm, but that was already the case afaik.

The pin that is assigned as P2, must also be D1. 

I think you mean you don't prefer option a, and I can switch A & B.